courses/free-courses-hi.md

```diff
@@ -159,9 +159,11 @@
 ### Databases
 
 * [DBMS (Database Management System)Complete Playlist](https://www.youtube.com/playlist?list=PLxCzCOWd7aiFAN6I8CuViBuCdJgiOkT2Y) - Gate Smashers
+* [DBMS Database Managament System- Complete Course in Hindi](https://youtube.com/playlist?list=PLmXKhU9FNesR1rSES7oLdJaNFgmuj0SYV) - Knowledge Gate
 * [DBMS Placements Series 2022](https://www.youtube.com/playlist?list=PLDzeHZWIZsTpukecmA2p5rhHM14bl2dHU) - CodeHelp by Babbar
 
 
+
```

Comment:
```suggestion
```

Reply:
Dont mess with the spaces

Reply:
@metabiswadeep : Fixed the spacing issue